dcos_test_utils/aws.py

```diff
@@ -485,7 +485,7 @@ def get_host_ips(self):
     ),
 }
 
-
+# Update these mappings to expand OS support
```

Comment:
I didn't mean it as a TODO, more as a : # If you must expand OS support, update these mappings